.github/workflows/tests.yml

```diff
@@ -101,9 +101,10 @@ jobs:
       - uses: actions/checkout@v2
       - uses: jakebailey/pyright-action@v1
```

Comment:
Does the version need updating? Or does v1 already point to some version that supports `no-comments`?

Reply:
I pushed a v1.0.1 tag, then retagged v1 to point at the latest version; this is common practice for github actions (as much as I hate deleting tags). I may switch this to a v1 branch in the future depending on how much churn/support it gets.

Reply:
Nevermind, it is included in `@v1` https://github.com/jakebailey/pyright-action/commits/v1